Crash when clicking compose button in message list widget

**Describe the bug**
Clicking the envelope icon in the message list widget crashes the app.

**To Reproduce**
Steps to reproduce the behavior:
1. Set up an account
2. Drag the message list widget to a home screen
3. Click the envelope icon (compose message) in the upper right of the home screen widget
4. Observe app crash

**Environment (please complete the following information):**
 - K-9 Mail version: 5.734-SNAPSHOT

**Logs**
```
Caused by: java.lang.NullPointerException: Parameter specified as non-null is null: method kotlin.jvm.internal.Intrinsics.checkNotNullParameter, parameter accountUuid
   at com.fsck.k9.Preferences.getAccount(Unknown Source:2)
   at com.fsck.k9.activity.MessageCompose.onCreate(MessageCompose.java:284)
```

--- a/app/ui/legacy/src/main/java/com/fsck/k9/activity/MessageCompose.java
+++ b/app/ui/legacy/src/main/java/com/fsck/k9/activity/MessageCompose.java
@@ -281,7 +281,9 @@ public class MessageCompose extends K9Activity implements OnClickListener,
                 relatedMessageReference.getAccountUuid() :
                 intent.getStringExtra(EXTRA_ACCOUNT);
 
-        account = preferences.getAccount(accountUuid);
+        if (accountUuid != null) {
+            account = preferences.getAccount(accountUuid);
+        }
 
         if (account == null) {
             account = preferences.getDefaultAccount();

